docs/app/pages/css/css-sections/buttons/colored-buttons/colored-buttons.component.html

```diff
@@ -25,4 +26,40 @@
 
 <p>Buttons are <code>inline-block</code> level elements so whitespace in the HTML of the page is significant and will affect their layout.</p>
 
-<uxd-snippet [content]="snippets.compiled.sampleHtml"></uxd-snippet>
\ No newline at end of file
+<uxd-snippet [content]="snippets.compiled.sampleHtml"></uxd-snippet>
+
+<br>
+
+<h3>Focus On Keyboard Control Only</h3>
```

Comment:
Can you make this a separate documentation section which we can link directly to?